### ISSUE TITLE
Directly install pytorch (without cpu version) could introduce cuda warnings. 

### ISSUE THREAD

**Issue by shanyu-sys** [intel/analytics-zoo#387]
Our document in [orca-pytorch-quickstart](https://analytics-zoo.readthedocs.io/en/latest/doc/Orca/QuickStart/orca-pytorch-quickstart.html) suggests user to install torch with
```bash
pip install torch==1.7.1 torchvision==0.8.2
```
However it could introduce warnings below during `loss.backward()` since the default torch is with cuda.
```
/home/arda/anaconda3/envs/zoo-shan/lib/python3.6/site-packages/torch/autograd/__init__.py:132: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /pytorch/c10/cuda/CUDAFunctions.cpp:100.)
  allow_unreachable=True)  # allow_unreachable flag
```
